packages/manager/apps/vrack-services/src/pages/subnets/components/SubnetDataGridCells.tsx

```diff
@@ -31,13 +32,23 @@ export const DisplayNameCell: React.FC<DataGridCellProps<
 > & {
   vrackServices?: VrackServices;
   updateVS: UpdateVS;
-  trackEvent: (data: any) => Promise<void>;
-}> = ({ cellData, rowData, updateVS, vrackServices, trackEvent }) => (
+  trackEvent: (data: TrackingProps) => PromiseLike<void>;
+  trackClick: (prop: TrackingProps) => PromiseLike<void>;
```

Comment:
Maybe we can import them directly without use props ?

Reply:
No we can't, it's inside a cell of ODS datagrid for now